docs/uploading_and_publishing/explanation_how_the_processing_works.md

```diff
@@ -0,0 +1,158 @@
+
+We recommend you to keep as much auxiliary files as possible together with the mainfile, but without reaching the [uploads limits](refs.md/#uploads-limits). Please, also check our recommendations on [Best Practices: preparing the data and folder structure](refs.md/#best-practices-preparing-folder-upload).
+
+
+## Structured data with the NOMAD metainfo
+
+Once the mainfile and auxiliary files have been recognized, a new [entry](../glossary/glossary.md/#entries) in NOMAD is created and a specific [parser](#parsing) is executed. You can check more details in [Writing a parser plugin](../writing_a_parser_plugin/parser_plugin_overview.md) on how to add new parsers in order for NOMAD to support new codes.
+
+In this new entry, NOMAD generates a **NOMAD archive** section. This will contain all the (meta)information extracted from the unstructured raw data files but in a _structured_, _well defined_, and _machine readable_ format. We define the **NOMAD metainfo** as all the set of [sections, sub-sections](../glossary/glossary.md/#metadata-sections), and [quantities](../glossary/glossary.md/#metadata-quantities) used to structure the raw data into a structured _schema_. You can further read in the general NOMAD documentation page in [Learn > Structured data](https://nomad-lab.eu/prod/v1/staging/docs/learn/data.html).
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="/assets/uploading_and_publishing/nomad_metainfo.png" alt="The NOMAD metainfo." width="90%" title="The NOMAD metainfo.">
+    </label>
+</div>
+
+
+## NOMAD sections for computational data
+
+Under the entry `archive` section, there are several main sections being populated by the parsers. For computational data, the only populated ones are:
+
+- `metadata`: contains general and non-code specific metadata. This is mainly information about authors, creation of the entry time, identificators (id), etc.
+- `run`: contains the **parsed** and **normalized** raw data into the structured NOMAD schema. This is all the possible raw data which can be translated into a structured way.
+- `workflow2`: contains metadata about the specific workflow done in the entry. This is mainly a set of well-defined workflows, e.g., `GeometryOptimization`, and their parameters.
+- `results`: contains the **normalized** and **indexed** metadata. This is mainly linked with searching, filtering, and visualizing data in NOMAD.
+
+??? question "`workflow` and `workflow2` sections: refactoring and deprecating"
+    You have probably noticed the name `workflow2` but also the existence of a section called `workflow` under `archive`. This is because
+    `workflow` is an old version of the workflow section, while `workflow2` is the newest one. `workflow` is still present in some of the older entries, so 
+    that until the full NOMAD repository and its entries are reprocessed with the new section `workflow2`, we need to maintain the old section.
```

Comment:
This is a more general comment. It is of such importance that I would even put it in a highlighted box.
Maybe it should go under a section Requesting new features / reporting bugs?

Reply:
Here is good enough. It has the question folded thingy, which I think is quite appealing and will make people say "ahh, ok".